x-pack/plugins/cloud_security_posture/public/pages/findings/findings_container.tsx

```diff
@@ -121,22 +121,24 @@ export const FindingsContainer = ({ dataView }: { dataView: DataView }) => {
           />
         )}
         <EuiSpacer />
-        <FindingsDistributionBar
-          total={findingsResult.data?.total || 0}
-          passed={countResult.data?.passed || 0}
-          failed={countResult.data?.failed || 0}
-          pageStart={findingsQuery.from + 1} // API index is 0, but UI is 1
-          pageEnd={findingsQuery.from + findingsQuery.size}
-        />
-        <EuiSpacer />
         {groupBy === 'none' && (
-          <FindingsTable
-            {...findingsQuery}
-            setQuery={setUrlQuery}
-            data={findingsResult.data}
-            error={findingsResult.error}
-            loading={findingsResult.isLoading}
-          />
+          <>
+            <FindingsDistributionBar
```

Comment:
this will change later on once we add data to the new table which will use this distribution bar too. for now it needs to be hidden as it reflects only results of group by `none`